cli/internal/scope/scope.go

```diff
@@ -90,15 +104,17 @@ func ResolvePackages(opts *Opts, scm scm.SCM, ctx *context.Context, tui cli.Ui,
 	return filteredPkgs, isAllPackages, nil
 }
 
-func (o *Opts) getPackageChangeFunc(scm scm.SCM, packageInfos map[interface{}]*fs.PackageJSON) scope_filter.PackagesChangedSince {
+func (o *Opts) getPackageChangeFunc(scm scm.SCM, cwd string, packageInfos map[interface{}]*fs.PackageJSON) scope_filter.PackagesChangedSince {
+	// Note that "since" here is *not* o.Since. Each filter expression can have its own value for
+	// "since", apart from the value specified via --since.
```

Comment:
Optional: maybe `packageSince`?

Reply:
Changed the argument to `filterSince`. That seemed like a better fit, it is the git ref from a `--filter=` argument. I also updated the comment, there is no longer an `o.Since`, it is now `o.LegacyFilter.Since`, which I hope is more difficult to confuse.